cPP_DSC.adoc

```diff
@@ -783,137 +791,133 @@ The following table provides the allowed choices for completion of the selection
 
 |KAS2
 |RSA
-|{empty}[*selection*: [.underline]#2048, 3072, 4096, 6144, 8192]# bits
+|[selection: 2048, 3072, 4096, 6144, 8192] bits
 |NIST SP 800-56B Rev 2 (Section 8.3)
 
 |DH
 |Diffie-Hellman
-|{empty}[*selection*: [.underline]#2048, 3072, 4096, 6144, 8192]# bits
-|NIST SP 800-56A Rev 3, {empty}[*selection*: RFC 3526 (Section [.underline]#[*selection*: 3, 4, 5, 6, 7]), RFC 7919 (Appendixes [*selection*: A.1, A.2, A.3, A.4, A.5]#)]
+|[selection: 2048, 3072, 4096, 6144, 8192] bits
+|NIST SP 800-56A Rev 3, [selection: RFC 3526 (Section [selection: 3, 4, 5, 6, 7]), RFC 7919 (Appendixes [selection: A.1, A.2, A.3, A.4, A.5])]
 
 |ECDH-NIST
 |ECDH with NIST curves
-|{empty}[*selection*: [.underline]#256 (P-256), 384 (P-384), 512 (P-521)]#
-|NIST SP 800-56A
+|[selection: 256 (P-256), 384 (P-384), 512 (P-521)] bits
+|NIST SP 800-56Ar3, NIST SP 800-186 (Appendix G.1)
 
 |ECDH-BPC
 |ECDH with Brainpool curves
-|{empty}[*selection*: [.underline]#256 (brainpoolP256r1), 384 (brainpoolP384r1), 512 (brainpoolP512r1)]#
-|RFC 5639 (Section 3)
+|[selection: 256 (brainpoolP256r1), 384 (brainpoolP384r1), 512 (brainpoolP512r1)] bits
+|NIST SP 800-56Ar3, NIST SP 800-186 (Appendix H.1)
 
 |ECDH-Ed
 |ECDH with Edwards Curves
-|{empty}[*selection*: [.underline]#256, 448]# bits
+|[selection: 255, 448] bits
```

Comment:
I copied this from the catalog, hence the change. I think there was a comment in the catalog about it. I don't have an opinion. @slpotte do you have anything to say here?

Reply:
I don't see any comment in the catalog about this specific key size

Reply:
What I can say is that we deliberately took out references to key sizes in asymmetric algorithms.   We want to replace references to "key sizes" in asymmetric algorithms to cryptographic parameter domains.  In the case of elliptic curves, once the curve is chosen, this fixes all other parameters with the exception of the private key itself.  The catalog provides a vendors and schemes with a list of allowed algorithms and where appropriate, choices within those algorithms that correspond with security strengths that the schemes want.  In FCS_COP.1/SigVer, the "Cryptographic Key Size" column should be labeled "Cryptographic Algorithm Parameters".

Reply:
So I think it makes sense to change some of the tables to have "Cryptographic Algorithm Parameters" but that doesn't answer the question about what the right key size should be here. I am editing the table header as well as the SFR to list "Cryptographic Algorithm Parameters" in the key sizes selection (since that is now where you would find the key size).

I will check the remaining tables for consistency as well.

Reply:
As a note, any table that had key sizes but had more than just sizes was changed to "Cryptographic Algorithm Parameters". Otherwise I left it alone if the key size was the only thing in the column.

Reply:
I think this table and the KDF table are the only tables that have a key sizes header. The KDF table doesn't have any asymmetric algorithms, so it's not an issue there. I agree with @slpotte though that it is easiest to just list the curve/modulus size/... rather than the key sizes for asymmetric algorithms.

Reply:
@woodbe since you changed the header to "Cryptographic Algorithm Parameters", will you also change the contents?

Reply:
What is the right answer? Is it just 256 instead of 255? That wasn't clear to me. I can change it, but not sure what the right answer is (that is not my area of expertise)

Reply:
@woodbe @slpotte I made the update to the cPP here: https://github.com/DSC-iTC/cPP/pull/308/commits/0ac82162cee63f468f6d2281575a0b92e9c59017. Let me know what you think.

Reply:
wait, I am looking at this later without paying attention. resolved.